Sources/TuistDependencies/Carthage/Utils/CarthageController.swift

```diff
@@ -60,13 +60,15 @@ public protocol CarthageControlling {
     /// - Parameters:
     ///   - path: Directory where project's dependencies will be installed.
     ///   - platforms: The platforms to build for.
-    func bootstrap(at path: AbsolutePath, platforms: Set<TuistGraph.Platform>) throws
+    ///   - printOutput: When true it prints the Carthage's ouput.
+    func bootstrap(at path: AbsolutePath, platforms: Set<TuistGraph.Platform>, printOutput: Bool) throws
```

Comment:
If we are using true everywhere, is it worth it to make the interface more complex adding another parameter?
We could:
- Don't pass any parameter and always print the output
- Pass the parameter to the constructor instead of the methods, given the info is needed by all methods